notion/package.json

```diff
@@ -7,7 +7,7 @@
   "author": "",
   "license": "Apache-2.0",
   "dependencies": {
-    "@gptscript-ai/gptscript": "github:gptscript-ai/node-gptscript#b30db3cf4fe87b1ee09a20752baf2263b1e5c7be",
+    "@gptscript-ai/gptscript": "github:g-linville/node-gptscript#49125f86e4a0c2af02b85373fdf647699dfcd58b",
```

Comment:
I'd rather just wait until grant's changes are merged rather than using his fork in the primary tool repo

Reply:
OK -- the dataset changes have merged and I updated this dependency.